src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/Text_Format.txt

```diff
@@ -1,98 +0,0 @@
-﻿>> Text(DateTime(1983, 6, 3, 2, 53, 9, 0), "utc") // utc1
-"1983-06-03T02:53:09.000Z"
-
->> Text(30470.519531251, "utc") // excelUtc1
-"1983-06-03T12:28:07.500Z"
```

Comment:
old v0 behavior, 

new v1 behavior, Should be in:
#SETUP: disable:PowerFxV1CompatibilityRules

Reply:
update to use StronglyTypedBuiltinEnums flag enable and disable.